app/subscriber.py

```diff
@@ -34,16 +34,23 @@ def receipt_to_case(message: Message):
             logger.error('Unknown Pub/Sub Message eventType', eventType=message.attributes['eventType'])
             return
         bucket_name, object_name = message.attributes['bucketId'], message.attributes['objectId']
-        logger.info('Pub/Sub Message received for processing',
-                    bucket_name=bucket_name,
-                    message_id=message.message_id,
-                    object_name=object_name)
-        payload = json.loads(message.data)
-        time_obj_created = parse_datetime(payload['timeCreated']).isoformat()
-    except KeyError:
-        logger.exception('Pub/Sub Message missing attribute')
+        payload = json.loads(message.data)  # parse metadata as JSON payload
+        metadata = payload['metadata']
+        case_id, tx_id, created = metadata['case_id'], metadata['tx_id'], payload['timeCreated']
+    except KeyError as e:
+        logger.error('Pub/Sub Message missing required data', missing=e.args[0])
```

Comment:
we prob should be logging something out here to enable use to link back to eq somehow